tests/MongoDB.Driver.Tests/RetryableWritesTests.cs

```diff
@@ -43,6 +45,34 @@ public void Insert_with_RetryWrites_true_should_work_whether_retryable_writes_ar
             }
         }
 
+        [SkippableFact]
+        public void Retryable_write_errorlabel_should_not_be_added_with_retryWrites_false()
+        {
+            RequireServer.Check().Supports(Feature.RetryableWrites).ClusterTypes(ClusterType.Sharded, ClusterType.ReplicaSet);
+
+            var failPointWithRetryableError = @"
+            {
+                configureFailPoint : 'failCommand',
+                mode : { times : 1 },
+                data : {
+                    failCommands : ['insert'],
+                    errorCode : 262
+                }
+            }"; // no error label
+
+            using (var client = GetClient(retryWrites: false))
+            using (ConfigureFailPoint(failPointWithRetryableError))
+            {
+                var database = client.GetDatabase(DriverTestConfiguration.DatabaseNamespace.DatabaseName);
+                var collection = database.GetCollection<BsonDocument>(DriverTestConfiguration.CollectionNamespace.CollectionName);
+                var document = new BsonDocument("x", 1);
```

Comment:
I would add a blank line between lines 68 and 69.

Line 68 is the last line of the Arrange section and line 69 is the Act section (and lines 71-72 are teh Assert section).